drools-parser/src/main/java/org/drools/parser/DRLVisitorImpl.java

```diff
@@ -577,16 +576,26 @@ public String visitDrlIdentifier(DRLParser.DrlIdentifierContext ctx) {
     @Override
     public ExistsDescr visitLhsExists(DRLParser.LhsExistsContext ctx) {
         ExistsDescr existsDescr = new ExistsDescr();
-        BaseDescr descr = visitLhsPatternBind(ctx.lhsPatternBind());
-        existsDescr.addDescr(descr);
+        if (ctx.lhsOrDef() != null) {
+            BaseDescr descr = visitDescrChildren(ctx.lhsOrDef()).get(0);
```

Comment:
@tkobayas 
Please check and manage the case where this -> `visitDescrChildren(ctx.lhsOrDef())` returns an empty list

Reply:
Thanks, I'll work on it.

Reply:
Done